test/test_determination.py

```diff
@@ -74,7 +74,7 @@ def test_test_file(self):
             ["distributed/rpc/test_process_group_agent"],
         )
         self.assertEqual(
-            self.determined_tests(["test/quantization/test_quantize.py"]),
+            self.determined_tests(["test/quantization/test_quantize_eager_ptq.py"]),
```

Comment:
Why is this specific file chosen instead of test_quantize?

Reply:
We deleted test_quantize in the PR, so I used another test for this. I think it doesn't matter which specific one we use here.